common/source/_static/common_theme_override.css

```
@@ -45,3 +45,14 @@ table.partners-table td {
     padding-top: 40px;
     margin-top: -40px;
 }
+
+/* make certain toc sidebar items red */
+.wy-menu .toctree-l1 a[href="docs/arduplane-setup.html"],
+.wy-menu .toctree-l1 a[href="docs/first-flight-landing-page.html"],
+.wy-menu .toctree-l1 a[href="docs/initial-setup.html"],
+.wy-menu .toctree-l1 a[href="docs/flying-arducopter.html"],
+.wy-menu .toctree-l1 a[href="docs/apmrover-setup.html"],
+.wy-menu .toctree-l1 a[href="docs/rover-first-drive.html"]
+{
+    color: red;
```

```suggestion
    color: red; font-weight: bold;
```
too bad it isnt persistent....also, what about the Quadplane TOC entries? can they be made to do the same thing?

---

Yep! Just need to add the correct CSS class and attribute things. What do you mean by persistent?

---

if you click on another menu item, and then return (ie close that items expansion) the style reverts....its only red until you click something....not a biggie

---

Fixed both